integrations/fedora.md

```diff
@@ -11,119 +11,119 @@ This page provides an overview of what you can do with the Fedora Linux Host int
 2. Click **Integrations** on the toolbar, search for and click the **Fedora Linux Host** tile. 
 3. Click the **Setup** tab and you will see the most recent and up-to-date instructions.
 
-## Linux Host Integration
-
-Monitoring Linux hosts is easy with Wavefront. This integration installs and configures Telegraf to send host metrics
-into Wavefront. Telegraf is a light-weight server process capable of collecting, processing, aggregating, and sending metrics to a [Wavefront proxy](https://docs.wavefront.com/proxies.html).
-
-In addition to setting up the metrics flow, this integration also installs a dashboard. Here's the CPU section of a dashboard displaying Linux host metrics.
-
-{% include image.md src="images/linux_db.png" width="80" %}
-
-
-## Linux Logs Setup
-
-This integration uses Fluentd input plugins and offers several setup options. For Fluentd installation prerequisites, see the [Fluentd Requirements](https://docs.fluentd.org/installation/before-install).
-
-* Use a proxy already running in your environment (preferred) or install a new proxy. 
-* Install the Wavefront proxy and Fluentd agent on the same host or on separate hosts.
-
-### Install the Wavefront Proxy
-
-The Wavefront proxy is required to send logs from your systems into Tanzu Observability. If you have not already done so, install a Wavefront proxy (version 11.4 or later).
-
-
-### Install Fluentd
-Install Fluentd to send logs data to Wavefront proxy.
-
-* See the [prerequisites](https://docs.fluentd.org/installation/before-install) for installing Fluentd.
-{% raw %}
-* To install the stable distribution package of Fluentd, called ```td-agent```, use one the following cURL commands, depending on the Linux distribution that you use:
-    
+## Linux Host Integration
+
+Monitoring Linux hosts is easy with Wavefront. This integration installs and configures Telegraf to send host metrics

```

Comment:
Monitoring Linux hosts is easy with VMware Aria Operations for Applications. This integration installs and configures Telegraf to send host metrics into our system.